Update description of intelliJ checkstyle configuration

marcelluethi opened the pull request:
The instructions for configuring checkstyle in intelliJ seem not to be up-to-date. This PR updates the description to match the latest checkstyle version (5.41.0) and intellij version (2020.2). 

This PR changes one screenshot, which I found to be in two versions in the repo (```docs/images``` and ```docs/.gibook/assets```). It seems that the former is not used anymore. However, as I don't understand the documentation processes fully, I did not remove it.

koppor commented on docs/getting-into-the-code/guidelines-for-setting-up-a-local-workspace.md:
```diff
@@ -175,17 +175,16 @@ Contributions to JabRef's source code need to have a code formatting that is con
+  8. Save settings by clicking "OK"
+  9. Your configuration should now look like this:  
 
       ![checkstyle settings](../.gitbook/assets/intellij-checkstyle-settings.png)
```
Here is the link to the image. I think, gitbook copies (instead of moving) all images to that directory during synchronization.

Can you just delete the other image?

marcelluethi replied:
I just deleted the unused image, together with two other images that I found were not used anywhere in the project.

koppor reviewed (commented):
Thank you for the update!

marcelluethi commented:
I just found that the description of how to ignore the buildSrc in Gradle has also changed. I have updated the description accordingly. 

All the instructions given in the guide should now be up-to-date.  

I am still having some problems to run it from Intellij using Intellij's runner (not all generated files are found by intellij). However, I suppose that this issue is not related to the instructions in this guide.

koppor commented:
Thank you for the update! We can discuss the IntelliJ issues in our gitter channel
